cpp/tests/binning/bin_tests.cu

```diff
@@ -404,16 +415,15 @@ TYPED_TEST_CASE(StringBinTestFixture, StringTypes);
 
 TYPED_TEST(StringBinTestFixture, TestStringData)
```

Comment:
This function is not typed test. A typed test is a test for various types, using `TypeParam` as the input type.